src/CommunityToolkit.Maui/Behaviors/PlatformBehaviors/Touch/GestureManager.shared.cs

```diff
@@ -109,7 +109,7 @@ internal async Task ChangeStateAsync(TouchBehavior sender, bool animated, Cancel
 			}
 			catch (TaskCanceledException ex)
 			{
-				Trace.WriteLine(ex);
+				Trace.TraceError("{0}", ex);
```

Comment:
Please do not use `TraceError` in the library. It causes crash reporting tools like sentry.io to report it as an unhandled exception. Leading to our library causing erroneous bug reports for our users.